python/pyspark/ml/classification.py

```diff
@@ -374,6 +415,48 @@ def getFamily(self):
         """
         return self.getOrDefault(self.family)
 
+    @since("2.2.0")
```

Comment:
Probably. I've seen that Scala version has been targeted for 2.2.1 so who knows? But let's make 2.3.